src/zrc/datamodel/models.py

```diff
@@ -329,6 +344,12 @@ class Meta:
         verbose_name = "Rol"
         verbose_name_plural = "Rollen"
 
+    def unique_representation(self):
+        if not hasattr(self, '__unique_representation'):
+            betrokkene_id = request_object_attribute(self.betrokkene, 'identificatie', 'betrokkene')
```

Comment:
The actual attribute (`identificatie` here) varies with the `betrokkene_type`:
* https://github.com/VNG-Realisatie/gemma-mock-overigeregistratiecomponenten/blob/master/src/orc/brp/models.py#L14 for `natuurlijk_persoon`
* https://github.com/VNG-Realisatie/gemma-mock-overigeregistratiecomponenten/blob/master/src/orc/datamodel/models.py#L48 for `niet_natuurlijk_persoon`

(and those are the only ones currently implemented)

Reply:
yeah we deviate in places to avoid copying data over all the time, probably safest to go with uuid for now and wait for other release candidate changes to make it have more meaning

Reply:
`Rol.unique_representation` is changed